simtools/model/telescope_model.py

```diff
@@ -457,21 +456,40 @@ def removeParameters(self, *args):
                 raise InvalidParameter(msg)
         self._isConfigFileUpdated = False
 
-    def addParameterFile(self, filePath):
+    def addParameterFile(self, parName, filePath):
         '''
         Add a file to the config file directory.
 
         Parameters
         ----------
+        parName: str
+            Name of the parameter.
         filePath: str
             Path of the file to be added to the config file directory.
         '''
+        if not hasattr(self, '_addedParameterFiles'):
+            self._addedParameterFiles = list()
+        self._addedParameterFiles.append(parName)
         shutil.copy(filePath, self._configFileDirectory)
-        return
+
+    def exportModelFiles(self):
+        ''' Exports the model files into the config file directory. '''
+        db = db_handler.DatabaseHandler()
+
+        # Removing parameter files added manually (which are not in DB)
+        parsToExportFiles = copy(self._parameters)
```

Comment:
This name is misleading (I think). These are essentially all the parameters, no? Then you simply remove the ones which are files added manually.
The list you give to `db.exportModelFiles` is of all of the parameters and in the `db_handler` you choose from them the file ones, right?
In that case, I would change the name of the variable to make it clearer.

Reply:
Yes, you are right. I renamed it to parsFromDB. It's ok, I think, because the scope of the variable is very small.